docs/source/libraries.rst

```diff
@@ -7,49 +7,53 @@ Here is where the Libraries API's descriptions are located:
-| DAC         | No          | No      |                    |
-+-------------+-------------+---------+--------------------+
-| GPIO        | Yes         | Yes     |                    |
-+-------------+-------------+---------+--------------------+
-| SDIO/SPI    | Yes         | Yes     |                    |
-+-------------+-------------+---------+--------------------+
-| I2C         | Yes         | Yes     |                    |
-+-------------+-------------+---------+--------------------+
-| I2S         | No          | Yes     | In Development     |
-+-------------+-------------+---------+--------------------+
-| Wi-Fi       | Yes         | Yes     |                    |
-+-------------+-------------+---------+--------------------+
-| Bluetooth   | Yes         | Yes     |                    |
-+-------------+-------------+---------+--------------------+
-| RMT         | Yes         | Yes     |                    |
-+-------------+-------------+---------+--------------------+
-| Touch       | Yes         | Yes     | ESP32 & ESP32-S2   |
-+-------------+-------------+---------+--------------------+
-| Timer       | Yes         | Yes     |                    |
-+-------------+-------------+---------+--------------------+
-| UART        | Yes         | Yes     |                    |
-+-------------+-------------+---------+--------------------+
-| Hall Sensor | Yes         | Yes     | Only for ESP32     |
-+-------------+-------------+---------+--------------------+
-| LEDC        | Yes         | Yes     |                    |
-+-------------+-------------+---------+--------------------+
-| Motor PWM   | No          | No      |                    |
-+-------------+-------------+---------+--------------------+
-| TWAI        | No          | No      |                    |
-+-------------+-------------+---------+--------------------+
-| Ethernet    | Yes         | Yes     | Only for ESP32     |
-+-------------+-------------+---------+--------------------+
-| USB         | Yes         | Yes     | Only for ESP32-S2  |
-+-------------+-------------+---------+--------------------+
++---------------+---------------+---------------+---------------+--------------------+
+| Peripheral    | ESP32         | ESP32-S2      | ESP32-C3      | Comments           |
++===============+===============+===============+===============+====================+
+| ADC           | Yes           | Yes           | Yes           |                    |
++---------------+---------------+---------------+---------------+--------------------+
+| Bluetooth     | Yes           | Not Supported | Yes           |                    |
++---------------+---------------+---------------+---------------+--------------------+
+| DAC           | No            | No            | No            | Planned            |
++---------------+---------------+---------------+---------------+--------------------+
+| Ethernet      | Yes           | Not Supported | Not Supported |                    |
```

Comment:
I think SPI ETH is supported on S2 and C3. let's check

Reply:
But we are referring to the internal peripheral only. We can add a note or just keep the internal ones.
What do you think about it?

Reply:
PTAL